dart/lib/src/protocol/sentry_event.dart

```diff
@@ -238,29 +260,7 @@ class SentryEvent {
     }
 
     if (exception != null) {
-      json['exception'] = [
-        <String, dynamic>{
-          'type': '${exception.runtimeType}',
-          'value': '$exception',
-        }
-      ];
-      if (exception is Error && exception.stackTrace != null) {
-        json['stacktrace'] = <String, dynamic>{
-          'frames': encodeStackTrace(
-            exception.stackTrace,
-            origin: origin,
-          ),
-        };
-      }
-    }
-
-    if (stackTrace != null) {
-      json['stacktrace'] = <String, dynamic>{
-        'frames': encodeStackTrace(
-          stackTrace,
-          origin: origin,
-        ),
-      };
+      json['exception'] = [exception.toJson()];
```

Comment:
the exception doesn't need to be a list because of Dart, theres no nested exceptions, but we need to render in a list way because of serialization on the other native SDKs.
eg:
exception {values [ {exception...}]}
see workaround here https://github.com/getsentry/sentry-dart/pull/158/

Reply:
✅